vta/hardware/intel/scripts/soc_system.tcl

```diff
@@ -33,7 +33,156 @@ set_instance_parameter_value clk_0 {clockFrequencyKnown} {1}
 set_instance_parameter_value clk_0 {resetSynchronousEdges} {NONE}
 
 add_instance hps_0 altera_hps 18.1
+set_instance_parameter_value hps_0 {ABSTRACT_REAL_COMPARE_TEST} {0}
```

Comment:
I personally feel like this is unnecessary and not a good use of cognitive time and extra test time after the changes  :-)
But if you want I don't mind you to remove the extra lines in a different PR.

Reply:
Thanks again for pushing this through @pasqoc , I think it's fine to have follow-up PRs on this.

Reply:
The size of this generated script increased from 7 KB to 47 KB. What's your opinion on this @tmoreau89 ?

Reply:
This file is 760 lines of generated code with a few local modifications, it is not large by any standard and perfectly suitable for github. Apart from the very little return on investment in reducing it if one puts such a high value to file size, doing it seems counter productive. As in the case of this PR, next time one regenerates the script a whole manual diff needs to be done and different versions of the generating tool or even different runs with new components will change the layout of the file. It is no trivial to figure out what is needed and what not and any modifications need to be tested with full spins of bitstream generation and testing. Actually, a robust and agile way for doing this would be to save both the original untouched generated file and the applied patch, so that the patch can be quickly reapplied when the base script is regenerated. Modifying the generated script every time comparing to a manually reduced version has the potential of adding oversights that may be difficult to catch. So all in all, I do not see the benefits of reducing the file size maybe to 10KB? Especially since there are already so many other files larger than this one in the repo.